projects/elife.yaml

```diff
@@ -2227,6 +2227,7 @@ kubernetes-aws:
                 # but not the ExternalDNS chart release
                 external-dns:
                     domain-filter: "elifesciences.org"
+                autoscaler-policy: true
```

Comment:
When introducing new keys ensure they are mentioned in the `defaults` section with a sensible default. Add a comment if it's purpose isn't obvious.

Reply:
new value should probably be mirrored in the `./src/tests/fixtures/projects/dummy-project.yaml` file as well

Reply:
Thanks, added. Also added a new test fixture project and a test for the output while in there.